src/App.css

```diff
@@ -1,38 +1,58 @@
-.App {
-  text-align: center;
+.Nav-bar {
+  display: flex;
+  flex-direction: row;
```

Comment:
Please remove this declaration line as the `flex-direction` property default value is always `row` :smile:

Reply:
ayy my bad
but its not harmful :)